docs/guides/migrating-react-router-app.md

```diff
@@ -174,9 +174,9 @@ Lastly, in your root `App` component (the one that would have been mounted to th
 
 ## Creating a catch-all route
 
-Remix needs routes beyond the root route to know what to render in `<Outlet />`. Fortunately you already render `<Route>` components in your app, and Remix can use those as you migrate to use our [routing conventions](./routing.md).
+Remix needs routes beyond the root route to know what to render in `<Outlet />`. Fortunately you already render `<Route>` components in your app, and Remix can use those as you migrate to use our [routing conventions][routing-conventions].
 
-To start, create a new directory in `app` called `routes`. In that directory, create a file called `$.jsx`. This is called [a **catch-all route**](./routing#splats) and it will be useful to let your old app handle routes that you haven't moved into the `routes` directory yet.
+To start, create a new directory in `app` called `routes`. In that directory, create a file called `$.jsx`. This is called [a **catch-all route**][a] and it will be useful to let your old app handle routes that you haven't moved into the `routes` directory yet.
```

Comment:
Probably should handle this better.

Reply:
ah yup, i can broaden my "aggregate" function to accept all sorts of things https://github.com/mcansh/remark-definition-links/blob/main/src/index.ts#L8

Reply:
ah yup, i can broaden my "aggregate" function to accept all sorts of things https://github.com/mcansh/remark-definition-links/blob/main/src/index.ts#L8